### PR TITLE
Actually update confirm email language in template

### PR DESCRIPTION
**Pull request by erinspace**
PR to amend #5019

Related to https://openscience.atlassian.net/browse/OSF-5697
(Not sure how this happened, sorry!!)

# Purpose
Builds off of already merged PR that somehow left out the essential language update... Pretty simple, and actually changes the language when it wasn't before. Oops!

# Changes
Update confirm email template

### CODE REVIEW

File: website/templates/emails/confirm.txt.mako [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Hello ${user.fullname},
 
-Welcome to the Open Science Framework. Please verify your email address by visiting this link:
+This email address has been added to an account on the Open Science Framework.
```
Comment (chennan47) [inline website/templates/emails/confirm.txt.mako]: I think it will be better if after "an account" you can tell which account it has been added to.

**Comment by chennan47** [comment]
Some language suggestion for discussion. Otherwise, good for merge.

**Comment by erinspace** [comment]
@chennan47 how should the account be identified? With the primary email address?

**Comment by chennan47** [comment]
@erinspace It can be primary email address or a link to the profile page. Might ping community team for decision.

**Comment by erinspace** [comment]
@nanchen47 OK! Checked with community, they are wary of exposing email addresses, or additional links that may clutter the email. Since the user name is at the beginning of the email, this is a bit of an identifier. 

Also, build seems to be failing for unrelated reasons (this is a simple text change) can anyone with power restart it?

**Pull request merged**